src/renderers/dom/shared/__tests__/CSSPropertyOperations-test.js

```diff
@@ -60,12 +60,12 @@ describe('CSSPropertyOperations', function() {
     })).toBe('left:0;margin:16px;opacity:0.5;padding:4px;');
   });
 
-  it('should trim values so `px` will be appended correctly', function() {
+  it('should trim values', function() {
     expect(CSSPropertyOperations.createMarkupForStyles({
-      margin: '16 ',
+      left: '16 ',
```

Comment:
No weirder… you would probably never realistically set `left` to a unitless 16 either.